addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -453,11 +449,7 @@ odoo.define('website.tour.form_editor', function (require) {
         // B) Add a 3rd field. Field A's visibility is tied to field B being set,
         // field B is autopopulated and its visibility is tied to field C being
         // set, and field C is empty.
-        {
-            content: 'Enter in edit mode again',
-            trigger: 'a[data-action="edit"]',
-            run: 'click',
-        },
+        ...wTourUtils.clickOnEditAndWaitEditMode(),
```

Comment:
You can then remove `extra_trigger: 'button[data-action="save"]'` in next step to be correct